install/syndesis.yml

```diff
@@ -996,7 +999,6 @@ objects:
       syndesis.io/type: infrastructure
     annotations:
       console.alpha.openshift.io/overview-app-route: "true"
-      haproxy.router.openshift.io/timeout: 60s
```

Comment:
Any idea why this was added in the first place and now removed ?

Reply:
Seems to be added as a side effect of #4139 without adding it to the templates in `install/generator/`. @rkorytkowski if we need this it should be part of the templates in `install/generator/`

Reply:
It's no longer needed. I was able to get the processing time significantly lower than the 30s default.